.changeset/eighty-ladybugs-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Fix DEV mode on Safari when view transitioning to `client:only` components
```

Comment:
```suggestion
Fixes an error in dev mode on Safari where view transitions prevented navigating to pages with `client:only` components
```
That's for tapping me in, @martrapp! I could see something like above!

Reply:
Accepted! Thank you very much!